rust/operator-binary/Cargo.toml

```diff
@@ -6,27 +6,25 @@ edition = "2021"
 license = "OSL-3.0"
 name = "stackable-trino-operator"
 repository = "https://github.com/stackabletech/trino-operator"
-version = "0.0.0-dev"
 publish = false
+version.workspace = true
 
 [dependencies]
-stackable-operator = { git = "https://github.com/stackabletech/operator-rs.git", tag = "0.45.1" }
 stackable-trino-crd = { path = "../crd" }
 
-anyhow = "1.0"
-async-trait = "0.1"
-clap = "4.3"
-futures = { version = "0.3", features = ["compat"] }
-indoc = "1.0.7"
-openssl = "0.10.42"
-pin-project = "1.0"
-semver = "1.0"
-snafu = "0.7"
-strum = { version = "0.24", features = ["derive"] }
-tokio = { version = "1.24", features = ["full"] }
-tracing = "0.1"
+anyhow = { workspace = true }
+async-trait = { workspace = true }
+clap = { workspace = true }
+futures = { workspace = true }
+indoc = { workspace = true }
+openssl = { workspace = true }
+pin-project = { workspace = true }
+semver = { workspace = true }
+snafu = { workspace = true }
+stackable-operator = { workspace = true }
+strum = { workspace = true }
+tokio = { workspace = true }
+tracing = { workspace = true }
 
 [build-dependencies]
-built = { version =  "0.6", features = ["chrono", "git2"] }
-stackable-operator = { git = "https://github.com/stackabletech/operator-rs.git", tag = "0.45.1" }
-stackable-trino-crd = { path = "../crd" }
+built = { workspace = true }
```

Comment:
Is it intentional that you removed these two features? If yes, ignore this comment.

Reply:
Well never mind. They are now enabled workspace-wide...

Reply:
exactly :)